web/libs/editor/src/tags/custom/Custom.jsx

```diff
@@ -249,12 +239,18 @@ const Model = types
         const store = getRoot(self);
         const annotation = self.annotation;
         if (annotation.pk && annotation.exists) {
-          if (store && store.events) {
-            store.events.invoke("updateAnnotation", store, annotation).catch(() => {});
+          if (store && store.events && typeof store.events.invoke === "function") {
+            try {
+              const res = store.events.invoke("updateAnnotation", store, annotation);
+              if (res && typeof res.catch === "function") res.catch(() => {});
+            } catch (e) {}
           }
         } else {
-          if (store && store.events) {
-            store.events.invoke("submitAnnotation", store, annotation).catch(() => {});
+          if (store && store.events && typeof store.events.invoke === "function") {
+            try {
+              const res = store.events.invoke("submitAnnotation", store, annotation);
+              if (res && typeof res.catch === "function") res.catch(() => {});
+            } catch (e) {}
```

Comment:
we have to remove all these calls completely! result update should not submit annotation, that's totally wrong. also it looks like it doesn't work anyway.